www/include/options/accessLists/actionsACL/listsActionsAccess.php

```diff
@@ -36,20 +36,27 @@
 if (!isset($centreon)) {
     exit();
 }
-    
+
 include("./include/common/autoNumLimit.php");
+$num = (int) $num;
+$limit = (int) $limit;
```

Comment:
please check that limit > 0. If not set a default value

Reply:
all the checks are already done in autoNumLimit.php